beerus_core/Cargo.toml

```diff
@@ -20,3 +20,5 @@ hex = "0.4.3"
 log = "0.4.17"
 env_logger = "0.10.0"
 mockall = "0.11.3"
+starknet = { git = "https://github.com/xJonathanLEI/starknet-rs" }
```

Comment:
should we fix a version ? or do we expect to be stable enough to always use the latest version ?